packages/falcon-shop-data/src/Customer/IsAuthenticatedQuery.tsx

```diff
@@ -9,14 +9,15 @@ export const GET_IS_AUTHENTICATED = gql`
   }
 `;
 
-export type IsAuthenticatedQueryData = {
+export type IsAuthenticatedQueryResponse = {
```

Comment:
```suggestion
export type IsAuthenticatedResponse = {
```

maybe we could make it shorter. The same convention could be used for `mutations`.
Let say we have `Adderess`  and
* `query` `GET_ADDRESS_BY_ID` will return  `AddressResponse`
* `mutation` `ADD_ADDRESS` will return  `AddressResponse` too